website/benchmarks.html

```diff
@@ -153,10 +156,25 @@ <h3 id="syscalls">Syscall count <a href="#syscalls">#</a></h3>
     <script type="module">
       import { drawCharts } from "./app.js";
       window.chartWidth = 800;
+      const busy = document.getElementById("busy")
```

Comment:
I think we should name it more component-like way, like `spinner-wrapper`, `spinner-overlay`, or `spinner-area`

Reply:
Sure `spinner-overlay` sounds best to me.